site/content/en/docs/manual/advanced/annotation-with-skeletons/manual-skeletons.md

```diff
@@ -0,0 +1,141 @@
+| **2**  | **Add point**               | Use it to add Skeleton points to the **Drawing area** (**8**).                                                                        |
+| **3**  | **Click and drag**          | Use it to move points across the **Drawing area** (**8**).                                                                            |
+| **4**  | **Add edge**                | Use it to add edge on the **Drawing area** (**8**) to connect the points (**2**).                                                     |
+| **5**  | **Remove point**            | Use it to remove points. Click on **Remove point** and then on any point (**2**) on the **Drawing area** (**8**) to delete the point. |
+| **6**  | **Download skeleton**       | Use it to download created skeleton in .SVG format.                                                                                   |
+| **7**  | **Upload skeleton**         | Use it to upload skeleton in .SVG format.                                                                                             |
+| **7**  | **Drawing area**            | Use it as a canvas to draw a skeleton.                                                                                                |
+
+<!--lint enable maximum-line-length-->
+
+### Creating Skeleton task
+
+To create **Skeleton** task, do the following:
+
+1. Open **Configurator**.
+2. (Optional) Upload background image.
+3. In the Label name field, enter the name of the label.
+4. (Optional) [**Add attribute**](/docs/manual/basics/create_an_annotation_task/#add-an-attribute)
+   <br>Note: you can add attributes exclusively to each point,
+   for more information, see [**Configuring Skeleton points**](#configuring-skeleton-points)
+5. Use **Add point** to add points to the **Drawing area**.
+6. Use **Add edge** to add edges between points.
+7. Upload files.
+8. Click:
+   - **Submit & Open** to create and open the task.
+   - **Submit & Continue** to submit the configuration and start creating a new task.
+
+### Configuring Skeleton points
+
+You can add labels, attributes and change colour of any point of the skeleton.
+
+To do this, right-click on the skeleton point and select **Configure**:
+
+![](/images/image-skeleton-drawn-example.jpg)
+
+In the opened menu, you can change the point setting. It is similar to
+[adding labels and attributes of the regular task](docs/manual/basics/create_an_annotation_task/#create-a-task):
+
+![](/images/image-skeleton-point-setup.jpg)
+
+> **Note** that you cannot change skeleton configuration for an existing task/project.
+
+> You can copy/insert skeletons configuration from the **Raw** tab of the label configurator.
+
+## Annotation with Skeletons
+
+To annotate with **Skeleton**, do the following
+
+1. Open job.
+2. On the tools panel select **Draw new skeleton**.
+3. Select **Track** to track object or **Shape** to annotate
```

Comment:
Such words make me think that CVAT will do tracking the object for me automatically